plugins/node/opentelemetry-instrumentation-express/README.md

```diff
@@ -78,6 +78,31 @@ Express instrumentation has few options available to choose from. You can set th
 - `info: ExpressRequestInfo` containing the incoming Express.js request, the current route handler creating a span and `ExpressLayerType` - the type of the handling layer or undefined when renaming the root HTTP instrumentation span.
 - `defaultName: string` - original name proposed by the instrumentation.
 
+Note: In order to ignore whole traces that represent a given Express route, use
+the `ignoreIncomingRequestHook` option from
+`@opentelemetry/instrumentation-http` against the route path. Ideally, this
+shouldn't be necessary since spans should a have low cardinality and minimize
+interaction between instrumentation libraies but
+`@opentelemetry/instrumentation-express` renames the root span from
+`@opentelemetry/instrumentation-http` in order to get things in order.
+
+```js
+registerInstrumentations({
+  instrumentations: [
+    // Express instrumentation expects HTTP layer to be instrumented
+    new HttpInstrumentation({
+      ignoreIncomingRequestHook(req) {
+        // Ignore spans from static assets.
+        const isStaticAsset = !!req.url.match(/^\/static\/.*$/);
+        return isStaticAsset;
+      }
+    }),
+    new ExpressInstrumentation(),
+  ],
+});
+```
```

Comment:
> the markdown isn't correctly formated, could you fix it please ?
>
> *-- https://github.com/open-telemetry/opentelemetry-js-contrib/pull/1095#issuecomment-1191244963*

I'm not sure what you mean? Perhaps the auto-wrap at 80 characters? Is there a different preference?

For reference, it looks fine in the rendered view, https://github.com/open-telemetry/opentelemetry-js-contrib/blob/23ccd953f402aa1dd349c4304c64f25b55b8da18/plugins/node/opentelemetry-instrumentation-express/README.md

Reply:
@vmarchaud Friendly ping for another round of review to clarify this point ^

Reply:
Totally sorry for not seeing this, checkout https://github.com/open-telemetry/opentelemetry-js-contrib/runs/8022040414?check_suite_focus=true

Reply:
Ahh, the CI linting. Thank you!

Fixed up ✅